lib/arena.rb

```diff
@@ -1,2 +1,92 @@
+require "pry"
+require_relative '../lib/gladiator'
+require_relative '../lib/arena'
+
+require './lib/arena'
+require './lib/gladiator'
+
 class Arena
+  attr_accessor :name, :gladiators
+  def initialize(name)
+    @name = name.capitalize
+    @gladiators = []
+  end
+  def add_gladiator(gladiator)
+    @gladiators << gladiator
+    @gladiators = @gladiators.first(2)
+    # if @gladiators.length < 2
+    #   @gladiators << gladiator
+    # end
+  end
+  def glad_weapons
+  # elsif @gladiators.each do |glad|
+  #   if glad.weapon ==
+  the_weapons = {
+      "Spear" => "Trident" #Spear loses to Trident
+      "Club" => "Spear" #Club loses to Spear
+      "Trident" => "Club" #Trident loses to Club
+    }
+    # set up a hash to do comparisons?
+
+    @gladiators.each do |glad|
```

Comment:
This errors out when you fix the other errors I mentioned because `@gladiators` doesn't exist until an Arena has been created. You could test this out in the initialize method though, when an Arena has definitely already been created.